index.d.ts

```diff
@@ -0,0 +1,19 @@
+declare module 'ember-breadcrumb-trail/helpers/breadcrumb' {
+  import Helper from '@ember/component/helper';
+
+  export default class Breadcrumb extends Helper<{
+    Args: {
+      Positional: [title: string];
+      Named: Record<string, unknown>;
+    };
+    Return: void;
+  }> {}
+}
+
+declare module 'ember-breadcrumb-trail/helpers/breadcrumbs' {
+  import Helper from '@ember/component/helper';
+
+  export default class Breadcrumbs extends Helper<{
+    Return: { title: string; data: { route: string; model: string } }[];
```

Comment:
The data object is everything that's passed as a named argument into the `{{breadcrumb}}` helper, so the route and model keys aren't guaranteed.

Should it be something like this maybe?
```suggestion
    Return: { title: string; data: Record<string, unknown> }[];
```

Reply:
Yes you are correct--this was an oversight on my end.